### PR TITLE
Align ascii graphic in readme

### PR DESCRIPTION
**Pull request by prees1**
Trivial fix to properly align the ascii diagram in the readme.

### CODE REVIEW

**Comment by pshevtsov** [comment]
This is how it looks for me

image

**Comment by prees1** [comment]
Yea its not perfect and can't be. My goal was to improve how it renders here on the GitHub site.

Before:
image

After:
image

Does that make sense to you?

**Comment by pshevtsov** [comment]
Looks like the monospaced font you are using lacks some glyphs (that arrow/triangle one, I suppose), so it falls back to other font which has that glyph. And it seems to me that font is not monospaced, so its width is not the same, that's why the output gets corrupted for you.

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ When taking an _application-centric_ approach to cloud migration, getting the re
 │                            │                           ║                                       ║
 │  CPU, RAM, Storage etc.    │                           ║  - Single Source of Truth             ║
 │                            │   `tidal sync servers`    ║  - Server, Database, and              ║
-│                            │──────────────────────────▶║    Application Inventory              ║
+│                            │─────────────────────────▶║    Application Inventory              ║
```
Comment (pshevtsov) [inline README.md:33]:
```suggestion
│                            │──────────────────────────>║    Application Inventory              ║
```

**Comment by prees1** [comment]
oh great point! got it. Thanks @pshevtsov, now the rendering here on GitHub looks perfect
image

**Pull request merged**